web/packages/teleport/src/Login/useLogin.ts

```diff
@@ -74,6 +75,12 @@ export default function useLogin() {
     history.push(ssoUri, true);
   }
 
+  const [showMotd, setShowMotd] = useState(!!motd);
+
+  const acknowledgeMotd = () => {
+    setShowMotd(false);
+  };
```

Comment:
nit: i'd prefer using `function acknowledgeMotd() {}` over arrow func (if you don't need the features of an arrow func), simply b/c it's easier to read imo (and other func in this comp is defined that way)